cmd/manager/main.go

```diff
@@ -15,6 +15,25 @@ import (
 	"strings"
 	"time"
 
+	"github.com/fsnotify/fsnotify"
+	"github.com/go-logr/logr"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"go.elastic.co/apm"
+	"go.uber.org/automaxprocs/maxprocs"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp" // allow gcp authentication
+	"k8s.io/client-go/rest"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/cache"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
+
```

Comment:
nit: according to https://github.com/elastic/cloud-on-k8s/blob/master/CONTRIBUTING.md#user-content-format-your-code-and-manage-imports we only want two groups of imports

Reply:
That slipped my mind completely. Is there a particular reason for that though? Personally, I find that a separate group for "project imports" makes things a little bit easier to read -- especially when there are lots of imports like we do in `main.go`.

Reply:
I am not sure how the two-groups rule came to be. Maybe @barkbay recalls I believe he wrote the guide.

Reply:
Sorry, the rule was already existing when I joined, I don't know where it comes from. Imports are sorted alphabetically so I'm not sure there are strong arguments to use either 2 or 3 groups _(maybe when there is an alias ?)_. I think the code is already organized with 2 groups so that would an argument to stick with it.

If we want to move to 3 groups my understanding is that using `goimports` can automatically _(with some IDE integration)_ do that for us with `goimports --local "github.com/elastic/cloud-on-k8s"`

Reply:
Yes, I refreshed my setup with my personal configuration and accidentally pulled in the `goimports` config that automatically formatted my PRs in the new way. I have reverted it. 

I think we can easily run `goimports` on the whole code base to use a new convention if we wanted to. However, this is just my personal preference, and it's not fair to ask everybody to change their settings. So let's keep things as they are.